hazelcast-wm/src/main/java/com/hazelcast/web/AbstractIMapWrapper.java

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.web;
+
+import com.hazelcast.core.EntryListener;
+import com.hazelcast.core.IMap;
+import com.hazelcast.map.EntryProcessor;
+import com.hazelcast.query.Predicate;
+import com.hazelcast.util.EmptyStatement;
+
+import java.util.Collection;
+import java.util.Map;
+import java.util.Set;
+
+public class AbstractIMapWrapper<K, V> {
+    final IMap imap;
```

Comment:
seems you don't use this class in the code. Did you use this class in somewhere in the code or am i missing something?